test/tools/spec-runner/index.js

```diff
@@ -676,7 +680,7 @@ const kOperations = new Map([
       const encryptedFields = operation.arguments.encryptedFields;
       const session = maybeSession(operation, context);
       return db.dropCollection(collectionName, { session, encryptedFields }).catch(err => {
-        if (err.code !== MONGODB_ERROR_CODES.NamespaceNotFound) {
+        if (!err.message.match(/ns not found/)) {
```

Comment:
It's safer to match on strings here as when promoting or not we could be working with a code that is a JS number or `Int32` object. Strings are always strings.